includes/Story_Post_Type.php

```diff
@@ -312,7 +311,6 @@ public static function admin_enqueue_scripts( $hook ) {
 					'allowedMimeTypes' => self::get_allowed_mime_types(),
 					'allowedFileTypes' => self::get_allowed_file_types(),
 					'postType'         => self::POST_TYPE_SLUG,
-					'postThumbnails'   => $post_thumbnails,
```

Comment:
But this was just used to pass the current post thumbnail to `ConfigProvider`. `postThumbnails` is nowhere used anymore, so I don't see the need for it.

Reply:
Perhaps we should then either always add the theme support for story post, or use another field for this value -- this will be a mandatory field and always needs to be available.

Reply:
@spacedmonkey Doesn't look like this causes a failure even if the theme doesn't support thumbnails, leaving this as is.